### ISSUE TITLE
Login is failing on the macOS with fish shell by default

### ISSUE THREAD

**Issue by vzaritovsky** [Microsoft/vscode-tools-for-ai#4]
Trying to execute "AI: Azure ML - Login", getting this message
```sh
The terminal process command `/usr/local/bin/fish --init-file /Users/vlza/.vscode-insiders/extensions/ms-toolsai.vscode-ai-0.1.1/out/scripts/azMlTerminal.sh` failed to launch (exit code: 1)
```
So, it's trying to execute "shell" script via the fish shell. Should be executed with "sh -c" or something.

Let me know if more information is needed.

**Comment by visionalsky** [comment]
Thank you vzaritovsky, we will investigate and fix soon.

**Comment by qfyin** [comment]
Fixed prior to 0.1.6
